accelerators-home/modules/ROOT/pages/simplifying-assets.adoc

```diff
@@ -0,0 +1,66 @@
+= Considerations for Simplifying Accelerator Assets
+
+MuleSoft Accelerators were designed following an API-led approach to connectivity, which promotes a clean separation of application, process, and system integration layers while enabling a high level of reusability - particularly with System APIs. While this powerful approach remains valid today, and is highly recommended for organizations of all sizes to enable future growth while minimizing maintenance costs, there are some considerations that need to be taken into account for customers who want to simplify their integrations and architectural approach. 
+
+== Simplifying Accelerator Assets
+
+A consistent driving factor behind the design and implementation of use cases supported by the MuleSoft Accelerators has always been to keep the assets and corresponding solutions as simple as possible while still providing a solid, functional, and maintainable architecture for customers to build upon. This has been achieved through the following key solution design principles:
```

Comment:
This feels sort of market-y to me. Is there a way to streamline and simplify the text?

Reply:
Re: "This" as the subject - see the [Style Ref](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these%22).

Reply:
Typically, we try to avoid "customer" (and "user") - instead address the reader directly.

Reply:
This doesn't make sense to me: The design and implementation of use cases?

Reply:
Actually, I'm wondering if this even needs to be said. What is it that you want to convey to the user? What do THEY need to do their jobs?

The bullet list below seems to be about how MuleSoft Accelerators are designed.

Reply:
I agree that it may not be needed because it doesn't provide actionable info. I'll check with the doc author to see if he is okay with removing it.